ehealthid-rp/src/test/java/com/oviva/ehealthid/relyingparty/ws/HealthEndpointTest.java

```diff
@@ -12,9 +18,36 @@ void get() {
     var sut = new HealthEndpoint();
 
     // when
-    var res = sut.get();
+    var httpServerExchange = mock(HttpServerExchange.class);
+    var headers = mock(HeaderMap.class);
+    var sender = mock(Sender.class);
+
+    when(httpServerExchange.getResponseHeaders()).thenReturn(headers);
+    when(httpServerExchange.getResponseSender()).thenReturn(sender);
+    when(httpServerExchange.getRequestMethod()).thenReturn(HttpString.tryFromString("GET"));
+
+    sut.handleRequest(httpServerExchange);
+
+    // then
+    verify(httpServerExchange).setStatusCode(Status.OK.getStatusCode());
+  }
+
+  @Test
+  void methodNotAllowed() {
+    var sut = new HealthEndpoint();
+
+    // when
+    var httpServerExchange = mock(HttpServerExchange.class);
+    var headers = mock(HeaderMap.class);
```

Comment:
we can remove headers because there is no interaction with them when the method is not allowed